docs/specifications/tests/Nameserver-TP/nameserver13.md

```diff
@@ -0,0 +1,91 @@
+# NAEMSERVER13: Test for EDNS truncated response
+
+## Test case identifier
+
+**NAMESERVER13**
+
+## Objective
+
+EDNS is a mechanism to announce capabilities of a DNS implementation,
+and is now basically required by any new functionality in DNS such as
+DNSSEC ([rfc 6891]).
+
+[RFC 6891, section 7] states that an OPT record must be included
+in a truncated response if the query includes an OPT pseudo record.
+The request will be queried with the only EDNS OPT pseudo-RR flag set; i.e. the
+'DO' bit. 
+
+The query will be tested on a DNSSEC signed zone. This test will not give a
+valid result if the zone is not signed.  
+
+In addition, the EDNS buffer size should be set to 512, since most signed DNSKEY
+responses are bigger than 512 bytes. 
+
+The objective  of the query in this test  is to elicit a truncated response
+from the server.
```

Comment:
This section is about the objective of the Test Case. The details of the buffer size fits better in the steps, where an explanation is given.

Reply:
I feel that the three sentences in the objective gives a better description to the audience